src/seq/fasta.rl

```diff
@@ -0,0 +1,251 @@
+
+
+# FASTA sequence types
+
+@doc """
+Metadata for FASTA sequence records containing just a `description` field.
+""" ->
+type FASTAMetadata
+    description::String
```

Comment:
Just because it is impossible to distinguish non-existence from empty string.
Writer of FASTA will not be able to know whether a white space should be added after the sequence name.